frontend/src/service/Api.service.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthService } from './Auth.service';
+import { Err, ErrorService } from './Error.service';
+import { EnvService } from './Env.service';
+import { GraphQLService, IGraphQLError } from './GraphQL.service';
+import { Url } from '../entity/Url';
+
+interface IGraphQLSchemaQuery {
+  authQuery: IGraphQLSchemaAuthQuery;
+}
+
+interface IGraphQLSchemaAuthQuery {
+  URLs?: IGraphQLSchemaURL[];
+}
+
+interface IGraphQLSchemaURL {
+  alias: string;
+  originalURL: string;
+}
+
+const GET_USER_SHORT_LINKS_QUERY = `
+  query params($authToken: String!) {
+    authQuery(authToken: $authToken) {
+      URLs {
+        alias
+        originalURL
+      }
+    }
+  }
+`;
+
+export class ApiService {
+  private graphQLBaseURL: string;
+
+  constructor(
+    private authService: AuthService,
+    private errorService: ErrorService,
+    private envService: EnvService,
+    private graphQLService: GraphQLService
+  ) {
+    this.graphQLBaseURL = `${this.envService.getVal(
+      'GRAPHQL_API_BASE_URL'
+    )}/graphql`;
+  }
+
+  invokeGetUserShortLinksApi(offset: number, pageSize: number): Promise<Url[]> {
+    return new Promise((resolve, reject) => {
+      this.graphQLService
+        .query<IGraphQLSchemaQuery>(this.graphQLBaseURL, {
+          query: GET_USER_SHORT_LINKS_QUERY,
+          variables: { authToken: this.authService.getAuthToken() }
```

Comment:
```suggestion
          variables: { authToken: this.authService.getAuthToken() }
```
Let's move this to a separate variable on top of line 46.